### ISSUE TITLE
Remove the circle from the mobile nav?

### ISSUE THREAD

**Issue by makenosound** [icelab/berg#70]
Feels a bit heavy to me and I’m not sure it’s necessary (this is in its default state)

**Comment by andyicelab** [comment]
What do you reckon @jasminwong?

**Comment by jasminwong** [comment]
Yeah, it feels like it's competing with the 'berg logo. The circle could be shown on `:active` only.

**Comment by andyicelab** [comment]
Sounds reasonable. I will make it so.

**Comment by andyicelab** [comment]
Have removed the border and left the circle for the `hover`/`focus` effect.

**Comment by makenosound** [comment]
Coolies. Feels a bit awkward when it’s visible still, I think because of the alignment:

![image](https://cloud.githubusercontent.com/assets/4353/16070635/38f786a0-331a-11e6-992c-ecd38ae3fde3.png)

Would be nice to introduce it with a transition too, so it’s not so strong. And, while we’re in here, there are a couple of things around the close button that feel a bit off to me:

1. The × isn’t centered within the circle.
2. The × is really really fat and feels out of place with the rest of the design, could we use a lighter glyph?

**Comment by andyicelab** [comment]
Yep, I can address both of those and add a transition :)

Will wait until we decide on the final logo before getting those dimensions perfect. One in there now is just a temp job.

**Comment by andyicelab** [comment]
Hey @makenosound — have made a bunch of changes here. Mind having a quick look?

Relevant commits:
* Templates: https://github.com/icelab/berg/commit/79c1ff07eb9114690982f5a31c84942c08c70171
* Assets: https://github.com/icelab/berg-assets/commit/e7acba99d19df5de1be26f328b923c44d46f43b1

Changes:
* Added the new logo asset and resized a touch;
* Refactored the nav toggle and inlined some SVG assets for the control glyphs;
* Added a little `background-color` transition;
* Added a CSS component (bare-bones for now) for glyphs, which I'll flesh out more when we get to the other icon replacements.

And an image for alignment:

![berg-alignment](https://cloud.githubusercontent.com/assets/1016074/16103506/8e652e4c-33bc-11e6-945f-e07fdd496d8d.jpg)

:)

**Comment by makenosound** [comment]
@andyicelab 👏  Nice one. I think all those bits are resolved, minor comments:

* The baseline for `Icelab` is a bit low I think, we want the word to feel vertically centred.
* There’s a bit of Sketch junk in the SVGs which we could clean up with [svgo-gui](https://github.com/svg/svgo-gui).

**Comment by andyicelab** [comment]
Thanks @makenosund — will adjust the baseline and clean up those SVG assets.
